sycl/test-e2e/ESIMD/unified_memory_api/scatter_usm.cpp

```diff
@@ -0,0 +1,37 @@
+//==------- scatter_usm.cpp - DPC++ ESIMD on-device test ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===------------------------------------------------------------------===//
+// RUN: %{build} -o %t.out
```

Comment:
nit: not requirement, but some convenience note:
For such big tests with lots and lots of kernels it is more convenient to have `per-kernel` compilation.
It is less stress for JIT compiler (smaller kernels, rather than 1 huge module) and if there are any jit-build issues, then it helps to see on which particular test-case/kernel jit fails.

Reply:
Sorry, just forgot to do this, will fix